app/services/config_report_importer.rb

```diff
@@ -0,0 +1,9 @@
+class ConfigReportImporter < ReportImporter
+  def self.authorized_smart_proxy_features
+    ['Puppet', 'Chef Proxy']
```

Comment:
I think don't need `'Chef Proxy'`, since it's called `'Chef'` and it's defined by chef plugins. It should be dropped from seeds as well (saying it's out of scope for this PR is OK)